### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by Feresey**

### CODE REVIEW

File: api/beacon/beacon_eddystone.go [hunk 1]
```diff
@@ -4,22 +4,24 @@ import (
...
 )
 
 const eddystoneSrvcUid = "FEAA"
 
 const (
 	frameTypeUID byte = 0x00
-	frameTypeURL      = 0x10
-	frameTypeTLM      = 0x20
+	frameTypeURL byte = 0x10
+	frameTypeTLM byte = 0x20
```
Comment (Feresey) [inline api/beacon/beacon_eddystone.go:16]: It seems to me that you want to specify an explicit type for these constants. But GO will not set the same type to the constant above. They will be `const untyped int`.

изображение

изображение

File: api/properties_service.go [hunk 2]
```diff
@@ -102,10 +102,6 @@ func (p *DBusProperties) AddProperties(iface string, props bluez.Properties) err
...
 //RemoveProperties remove a property set
 func (p *DBusProperties) RemoveProperties(iface string) {
-	if _, ok := p.props[iface]; ok {
-		delete(p.props, iface)
-	}
-	if _, ok := p.propsConfig[iface]; ok {
-		delete(p.propsConfig, iface)
-	}
+	delete(p.props, iface)
+	delete(p.propsConfig, iface)
```
Comment (Feresey) [inline api/properties_service.go:106]: This is unnecessary check

File: go.mod [hunk 3]
```diff
@@ -4,13 +4,25 @@ go 1.14
...
 	github.com/spf13/viper v1.6.2
-	github.com/stretchr/testify v1.2.2
+	github.com/stretchr/testify v1.5.1
 	github.com/suapapa/go_eddystone v0.0.0-20190827074641-8d8c1bb79363
-	golang.org/x/sys v0.0.0-20190422165155-953cdadca894
+	golang.org/x/sys v0.0.0-20200331124033-c3d80250170d
+	golang.org/x/text v0.3.2 // indirect
+	gopkg.in/ini.v1 v1.55.0 // indirect
+	gopkg.in/yaml.v2 v2.2.8 // indirect
 )
```
Comment (Feresey) [inline go.mod:28]: I'm just use `go get -u -v -t ./...`

File: bluez/profile/adapter/adapter_discovery.go [hunk 4]
```diff
@@ -72,10 +77,11 @@ func (a *Adapter1) OnDeviceDiscovered() (chan *DeviceDiscovered, func(), error)
 			}
 
 		}
-	})()
+	}()
 
 	cancel := func() {
 		omSignalCancel()
+		<-done
 		close(ch)
```
Comment (Feresey) [inline bluez/profile/adapter/adapter_discovery.go:85]: I've got panic on write to closed channel.

File: api/beacon/beacon_eddystone.go (hunk 1 above)
Comment (muka) [reply api/beacon/beacon_eddystone.go:16]: Good hint, I supposed indeed Go was keeping the group type to `byte`

**Comment by muka** [comment]
Great work! Thank you :1st_place_medal:

File: api/beacon/beacon_eddystone.go (hunk 1 above)
Comment (Feresey) [reply api/beacon/beacon_eddystone.go:16]: if you use iota, the type will remain

**Pull request merged**